plugins/hybrid_ae_pkg/backend/quantum/pl/QNN1.py

```diff
@@ -3,7 +3,7 @@
 import pennylane as qml
 import pennylane.numpy as np
 
-from plugins.hybrid_ae_pkg.backend.quantum.pl.TwoQubitGate import add_two_qubit_gate
+from .TwoQubitGate import add_two_qubit_gate
```

Comment:
if `TwoQubitGate` is a python module, then it should use `snake_case`. This could be changed in this PR or add an issue.